### PR TITLE
Use Tide to merge PRs in gitpod-io/ops

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description
This enables Tide for merging PRs in gitpod-io/ops once they have been reviewed. This is the first step towards gitpod-io/ops/issues/116. The overall aproach will be

1. Have prow merge PRs once they're reviewed (This is this PR)
2. Switch to PR based workflow for commits (https://github.com/gitpod-io/ops/pull/423)
3. Enable CODEOWNERS (upcoming PR in gitpod-io/ops)

## Related Issue(s)

Part of gitpod-io/ops/issues/116

## How to test

Tide should start merging PRs in gitpod-io/ops once they're approved.

## Release Notes

```release-note
NONE
```

## Documentation
N/A

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mads-hartmann** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by meysholdt — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 105304b7e1acb55b42097fddc5133f6b1f517b4c</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/59#pullrequestreview-826459767" title="Approved">meysholdt</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [meysholdt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
@mads-hartmann: Updated the `config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `config.yaml` using file `config/config.yaml`

<details>

In response to [this](https://github.com/gitpod-io/gitbot/pull/59):

>## Description
>This enables Tide for merging PRs in gitpod-io/ops once they have been reviewed. This is the first step towards gitpod-io/ops/issues/116. The overall aproach will be
>
>1. Have prow merge PRs once they're reviewed (This is this PR)
>2. Switch to PR based workflow for commits (https://github.com/gitpod-io/ops/pull/423)
>3. Enable CODEOWNERS (upcoming PR in gitpod-io/ops)
>
>## Related Issue(s)
>
>Part of gitpod-io/ops/issues/116
>
>## How to test
>
>Tide should start merging PRs in gitpod-io/ops once they're approved.
>
>## Release Notes
>
>```release-note
>NONE
>```
>
>## Documentation
>N/A


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
